baselines/bigquery-storage/package.json

```diff
@@ -30,7 +30,11 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.12.30",
     "c8": "^7.1.0",
-    "gts": "^1.1.2",
+    "gts": "next",
+    "eslint": "^6.8.0",
```

Comment:
All of the `eslint` dependencies are brought in by `gts` now.  You don't need them here :)

Reply:
removed `eslint` related dependencies, thx!

Reply:
They're still there though 😆

Reply:
Oh! I thought it was generator package.json, removed it now from template! :)